warehouse/query-core/src/test/java/datawave/query/cardinality/TestCardinalityWithQuery.java

```diff
@@ -65,6 +63,9 @@
     static String SHARD_TABLE_NAME = "shard";
     static String SHARD_INDEX_TABLE_NAME = "shardIndex";
```

Comment:
Not going to hold up an approval, while you're here you could change these to use TableName, which already has these constants defined.

Reply:
Done.